docs/source/release/v6.13.0/mantidworkbench.rst

```diff
@@ -7,22 +7,26 @@ Mantid Workbench Changes
 
 New Features
 ------------
-- The :ref:`Python script editor <WorkbenchScriptWindow>` now supports Bool syntax highlighting.
-- The :ref:`Python script editor <WorkbenchScriptWindow>` now supports f-string syntax highlighting.
-- Users can now save multiple selected workspaces as ASCII.
-- The crosshair button is disabled (invisible) in 3D plots, which include surface plot, wireframe plot and mesh plot.
-- The settings widget 'Apply' button is now disabled when there are no pending changes.
-- A crosshair toggle option has been added in mantidplots on the top right of the toolbar area. The crosshair button is disabled (invisible) in titled plots.
-- Deprecated parameter ``Unwrap Ref`` has been removed from the Powder
-  Diffraction Reduction Interface.
-- Added property ``errorreports.core_dumps``. Linux users can set this to the directory on their system where core dump files are put after a crash (e.g ``errorreports.core_dumps=/var/lib/apport/coredump``).
-  Workbench will then be able to use this property to extract useful information from the core dump file created after a crash and give that to the error reporting service.
-  This will help us to diagnose some problems where previously no stacktrace was available after a crash. On Linux, core dumps are now always turned on for the workbench process.
+- The :ref:`Python script editor <WorkbenchScriptWindow>` now supports ``bool`` and f-string syntax highlighting.
+- Users can now save multiple selected workspaces in the ASCII format.
+- A crosshair toggle option has been added in mantid plots on the top right of the toolbar area. The crosshair button is
+  disabled (invisible) in tiled and 3D plots.
+- The :ref:`WorkbenchSettings` widget's ``Apply`` button is now disabled when there are no pending changes.
+- Deprecated parameter ``Unwrap Ref`` has been removed from the Powder Diffraction Reduction Interface.
+- Added property ``errorreports.core_dumps``.
+
+  - Linux users can set this to the directory on their system where core dump files are put after a crash (e.g
+    ``errorreports.core_dumps=/var/lib/apport/coredump``). Workbench will then be able to use this property to extract
+    useful information from the core dump file created after a crash and give that to the error reporting service. This
+    will help us to diagnose some problems where previously no stacktrace was available
+    after a crash.
+  - On Linux, core dumps are now always turned on for the workbench process.
```

Comment:
The question will be - can Windows and OSX users be able to benefit from the Error Reporter change that allows C++ stack traces to be recorded. I appreciate this property can only be set in Linux.

Reply:
No. it does not work on windows or macos.